Specs/Flatglass.md

```diff
@@ -0,0 +1,90 @@
+# umati showcase information model for Flat Glass Processing
+
+In the following the umati showcase describes the additional information on top of the [OPC 40301 UA for Flat Glass Processing](https://www.vdma.org/viewer/-/v2article/render/15410388) companion specification.
+
+Please refer to the provided xml-Nodesets or UaModeler-project for exemplary implementation.
```

Comment:
@karstenroethig @Kantiran91 do you intend to provide also a UaModeler project?

Reply:
Copy-paste error. I have deleted it.